pkg/Library/NodeGraph/ConnectionUpdator.js

```diff
@@ -67,12 +67,13 @@ hasMatchingCandidate(connection, candidates) {
 updatePipelineConnections(pipeline, nodeTypes, candidates) {
   return values(pipeline.nodes)
     .map(node => this.updateNodeConnections(node, nodeTypes[node.type], candidates[node.id]))
-    .some(changed => changed);
+    .some(changed => changed)
+    ;
```

Comment:
nitnit - i'll try to configure my vscode to remove trailing whitespaces, but is it possible to avoid putting the semicolon on a separate line?

Reply:
I do that on purpose as it allows for the format below. Same for compound boolean expressions. I would not put the first expression on the same line as the `return` but it is a requirement for JS (!).

[operator][operand]
.
.
.
;

Reply:
The expression is explicitly terminated by the 'semi', but the individual clauses are easily moved around and are unencumbered by positional-punctuation.

This is why people sometimes format JSON with the commas _as prefixes to the next line_ (just looks bad IMO).